src/STEMMUS_SCOPE_exe.m

```diff
@@ -25,8 +25,15 @@ function STEMMUS_SCOPE_exe(config_file, runMode)
                        'SiteProperties', ... % Site properties (e.g. lat, lon)
                        'fluxes', ... % Atmospheric fluxes
                        'TT', ... % Soil temperature over depth
-                       'SoilVariables', ... % Structure that includes different variables of soil moisture, added by Mostafa
-                       'GroundwaterSettings' ... % added by Mostafa
+                       'SoilVariables', ... % Structure that includes different variables of soil moisture
+                       'GroundwaterSettings' ... % groundwater settings including input data from MODFLOW
+                       'gwfluxes' ... % structure that includes groundwater recharge and its individual components
+                       'EVAP' ... % evaporation
+                       'RWUs' ... % soil water root uptake
+                       'RWUg' ... % groundwater root uptake
+                       'R_Dunn' ... % Dunnian runoff
+                       'R_Hort' ... % Hortonian runoff

```

Comment:
The variables `R_Dunn` and `R_Hort` are only calculated after [this if statement](https://github.com/EcoExtreML/STEMMUS_SCOPE/blob/7614318f77e251c414768b0fe4078c24875121b1/src/STEMMUS_SCOPE.m#L626). However, the [save function of bmi](https://github.com/EcoExtreML/STEMMUS_SCOPE/blob/126d38e534e61d66b4f37fab4d7053a6d3ead331/src/STEMMUS_SCOPE.m#L727) fails if this variable does not exist in the workspace. Can we add this variable like an empty variable to the main script, e.g. after [Initialization routine](https://github.com/EcoExtreML/STEMMUS_SCOPE/blob/126d38e534e61d66b4f37fab4d7053a6d3ead331/src/STEMMUS_SCOPE.m#L38)?

Reply:
> The variables `R_Dunn` and `R_Hort` are only calculated after [this if statement](https://github.com/EcoExtreML/STEMMUS_SCOPE/blob/7614318f77e251c414768b0fe4078c24875121b1/src/STEMMUS_SCOPE.m#L626). However, the [save function of bmi](https://github.com/EcoExtreML/STEMMUS_SCOPE/blob/126d38e534e61d66b4f37fab4d7053a6d3ead331/src/STEMMUS_SCOPE.m#L727) fails if this variable does not exist in the workspace. Can we add this variable like an empty variable to the main script, e.g. after [Initialization routine](https://github.com/EcoExtreML/STEMMUS_SCOPE/blob/126d38e534e61d66b4f37fab4d7053a6d3ead331/src/STEMMUS_SCOPE.m#L38)?
>

Do you mean add two lines like below after the [Initialization routine](https://github.com/EcoExtreML/STEMMUS_SCOPE/blob/126d38e534e61d66b4f37fab4d7053a6d3ead331/src/STEMMUS_SCOPE.m#L38)?
`R_Dunn = 0`
`R_Hort = 0`